daml-lf/engine/src/test/scala/com/digitalasset/daml/lf/engine/CompiledPackageSpec.scala

```diff
@@ -1,183 +0,0 @@
-
-  val Seq(v1_6, v1_7, v1_8) =
-    Seq("6", "7", "8").map(minor =>
-      LanguageVersion(LanguageVersion.Major.V1, LanguageVersion.Minor.Stable(minor)))
-
-  private[this] final class ModBuilder(languageVersion: LanguageVersion) {
-
-    import ModBuilder._
-
-    val name = Ref.ModuleName.assertFromString(s"module${counter.incrementAndGet()}")
-
-    private[this] var _deps = Set.empty[PackageId]
-    private[this] var _body: Ast.Expr = AstUtil.EUnit
-    private[this] var _built = false
-
-    def addDeps(deps: (Ref.PackageId, Ref.ModuleName)): this.type = {
-      assert(!_built)
-      val (pkgId, modName) = deps
-      val id = Ref.Identifier(pkgId, Ref.QualifiedName(modName, noOpName))
-      _deps = _deps + id.packageId
-      _body = Ast.ELet(Ast.Binding(None, AstUtil.TUnit, Ast.EVal(id)), _body)
-      this
-    }
-
-    def build(): (Ast.Module, Set[PackageId]) = {
-      _built = true
-      Ast.Module(
-        name,
-        Map(noOpName -> Ast.DValue(AstUtil.TUnit, false, _body, false)),
-        languageVersion,
-        Ast.FeatureFlags.default
-      ) -> _deps
-    }
-  }
-
-  private[this] object ModBuilder {
-    private val counter = new AtomicInteger()
-
-    private def noOpName = Ref.DottedName.assertFromString("noOp")
-
-    def apply(
-        languageVersion: LanguageVersion,
-        deps: (Ref.PackageId, Ref.ModuleName)*,
-    ): ModBuilder =
-      deps.foldLeft(new ModBuilder(languageVersion))(_.addDeps(_))
-  }
-
-  private[this] final class PkgBuilder {
-
-    import PkgBuilder._
-
-    val id = Ref.PackageId.assertFromString(s"package${counter.incrementAndGet()}")
-
-    private[this] var _modules = List.empty[Ast.Module]
-    private[this] var _deps = Set.empty[PackageId]
-    private[this] var _built = false
-
-    def addMod(modBuilder: ModBuilder): this.type = {
-      assert(!_built)
-      val (mod, deps) = modBuilder.build()
-      _modules = mod :: _modules
-      _deps = _deps | deps
-      this
-    }
-
-    def build: Package = {
-      _built = true
-      Package(_modules, _deps, None)
-    }
-
-  }
-
-  private[this] object PkgBuilder {
-    private val counter = new AtomicInteger()
-
-    def apply(modBuilders: ModBuilder*): PkgBuilder =
-      modBuilders.foldLeft(new PkgBuilder())(_.addMod(_))
-  }
-
-  tests("PureCompiledPackages")(PureCompiledPackages(_).toOption.get)
-
-  tests("ConcurrentCompiledPackages") { packages =>
-    val compiledPackages = ConcurrentCompiledPackages()
-    packages.foreach {
-      case (pkgId, pkg) =>
-        compiledPackages
-          .addPackage(pkgId, pkg)
-          .consume(
-            _ => sys.error("unexpected error"),
-            packages.get,
-            _ => sys.error("unexpected error"),
-          )
-          .toOption
-          .get
-    }
-    compiledPackages
-  }
-
-  def tests(name: String)(_compile: Map[Ref.PackageId, Ast.Package] => CompiledPackages) =
-    s"$name#getMaxLanguageVersione" should {
```

Comment:
Just to be clear, you are killing this because of the other change to forbid dependencies on older LF versions not because the LF version moved from modules to packages, right?

Reply:
You are right. Let me kill this test in #7050